File: src/main/java/org/example/kalkulationsprogramm/repository/GewerkRepository.java
package org.example.kalkulationsprogramm.repository;

import org.example.kalkulationsprogramm.domain.Gewerk;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface GewerkRepository extends JpaRepository<Gewerk, Long> {
    List<Gewerk> findAllByOrderByNameAsc();
    List<Gewerk> findByAktivTrueOrderByNameAsc();
}

File: src/main/kotlin/org/example/kalkulationsprogramm/repository/GewerkRepository.kt
package org.example.kalkulationsprogramm.repository

import org.example.kalkulationsprogramm.domain.Gewerk
import org.springframework.data.jpa.repository.JpaRepository

interface GewerkRepository : JpaRepository<Gewerk, Long> {
    fun findAllByOrderByNameAsc(): List<Gewerk>

    fun findByAktivTrueOrderByNameAsc(): List<Gewerk>
}
